buildspec.yml

```diff
@@ -0,0 +1,22 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - ./scripts/hack/symlink-gopath-codebuild.sh
+      - cd /go/src/github.com/awslabs/amazon-ecr-credential-helper
+  pre_build:
+    commands:
+      - echo "Starting tests..."
+      - make test
+  build:
+    commands:
+      - echo "Starting build..."
+      - make
+  post_build:
+    commands:
+      - echo "Build completed on $(date)"
+artifacts:
+  files:
+      - '**/*'
```

Comment:
Does this grab the resulting build from the `./bin/local` directory and place it in `bin` ?

Reply:
Grabs all files under `bin` and places them in the S3 bucket.  Where it places them is configured by the build environment; you can see in my output above that they're placed in a namespaced folder (by the `BUILD_ID`).  It looks like `s3://<bucket>/<build_id>/amazon-ecr-credential-helper/local/docker-credential-ecr-login`.

Reply:
Now that this is rebased on top of #60, it produces artifacts for Linux, MacOS, and Windows.